### ISSUE TITLE
Update sample data

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse-api#270]
## Problem

The API has support for `filesize` and `filetype` fields after WordPress/openverse#266. These fields are not populated completely for image and audio samples.

| Media type | `filesize` | `filetype` |
|-|-|-|
| Image | ❌ | ❌ |
| Audio | ❌ | ✅ |

We also need to make sure these changes are reflected in the new data:

- https://github.com/WordPress/openverse-catalog/issues/263
- https://github.com/WordPress/openverse-catalog/issues/292

## Description

The presence of this data can help to ensure that these fields are working as expected.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by krysal** [comment]
From what I can tell, the current state of the sample data is as follows:

| Media type | `filesize` | `filetype` | `category` | 
|-|-|-|-|
| Image | ✅ (null) | ❌ | ❌ |
| Audio | ✅ | ✅ | ✅ |

Regarding the `audio_set_foreign_identifier` as a separate field in audio data, the change in the catalog is still pending so I'll leave that for another ticket and complete the fields for images as much as the sources allow.

**Comment by AetherUnbound** [comment]
The `audio_set_foreign_identifier` ticket is assigned to me in WordPress/openverse#1692, but I've been waiting on starting that pending a successful data refresh.
